.github/workflows/test.yml

```diff
@@ -33,13 +33,16 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       runner: ${{ steps.fork-check.outputs.runner }}
+      runner_virt: ${{ steps.fork-check.outputs.runner_virt }}
     steps:
       - id: fork-check
         run: |
           if [[ "${{ github.event.repository.full_name}}" == "lf-edge/eve" ]] || [[ "${{ github.event.repository.full_name}}" == "lf-edge/eden" ]]; then
             echo "runner=buildjet-4vcpu-ubuntu-2204" >> "$GITHUB_OUTPUT"
+            echo "runner_virt=[buildjet-4vcpu-ubuntu-2204, buildjet-pinned-7950x]" >> "$GITHUB_OUTPUT"
```

Comment:
Will it run on both types of machines (two separate workflows)? If so, please check that we will not have artifacts (archive with logs) wirh the same name.

Reply:
These two labels are ANDed to further narrow down suitable runner (see https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job#overview).
For the context, here is answer from Adam from Buildjet regarding failing nested virtualization:
```
Thank you for the many workflow examples! I think we figured it out. All the failing ones are using the AMD 5950x CPU and all the successful ones use the AMD 7950x CPU. As a solution, let's just pin your runners to the 7950x.

To pin your runners to the 7950x, simply convert the runs-on property to an array and add: buildjet-pinned-7950x. 
```

Reply:
Sorry, confused with strategy. And thank you for detailed response